s3tests/functional/test_headers.py

```diff
@@ -24,7 +23,7 @@
 from nose.plugins.skip import SkipTest
 
 from .utils import assert_raises
-import AnonymousAuth
+from . import AnonymousAuth
```

Comment:
for py2 probably `from __future__ import AbsoluteImport` should turn this behaviour on? (not so sure myself)

Reply:
and similarly `from __future__ import print_funciton`